tap_jira/streams.py

```diff
@@ -373,12 +366,18 @@ def sync_project(self, fieldNames, knownFields, project_key_or_id = None):
         last_updated = Context.update_start_date_bookmark(updated_bookmark)
         timezone = Context.retrieve_timezone()
         start_date = last_updated.astimezone(pytz.timezone(timezone)).strftime("%Y-%m-%d %H:%M")
-        if datetime.datetime(2024, 7, 15, 0, 0, tzinfo=pytz.utc) < last_updated < datetime.datetime(2024, 7, 23, 5, 0, 0, tzinfo=pytz.utc):
+        if datetime.datetime(2024, 7, 15, 0, 0, tzinfo=pytz.utc) < last_updated < datetime.datetime(2024, 8, 10, 5, 0, 0, tzinfo=pytz.utc):
             LOGGER.info('state is in broken timeframe, going back in time to ensure all issues are ingested')
             start_date = datetime.datetime(2024, 7, 15, 0, 0, tzinfo=pytz.utc).strftime("%Y-%m-%d %H:%M")
 
         issue_changelogs_updated_bookmark_path = [CHANGELOGS.tap_stream_id, project_key_or_id, "updated"]
         issue_changelogs_updated = Context.update_start_date_bookmark(issue_changelogs_updated_bookmark_path)
+        LOGGER.info('changelog updated state {}'.format(issue_changelogs_updated))
```

Comment:
This would be a lot to log for every issue -- probably don't want to include this when merging.

Reply:
Nevermind, just noticed it's only once per project, which seems fine.